### PR TITLE
Add Contact Form to Mobile Portfolio

### PR DESCRIPTION
**Pull request by jkanyi-web**
- Add contact form and update style.css file

### CODE REVIEW

**Comment by supreeti** [comment]
Add mobile view of contact form on my index.html page and made changes in style.css accordingly with collaboration of my learning partner.

File: style.css [hunk 1]
```diff
@@ -339,6 +339,108 @@
   margin: 15px 0;
 }
 
+.contact_section {
+  padding: 150px 0 20px 0;
+  border-top-left-radius: 100px;
+  display: flex;
+  justify-content: center;
+  background-color: #7f8cff;
```
Comment (leonardodiasb) [inline style.css:347]: - [ ] Your form works fine and looks similar to the Figma template, but the background color is a little bit different. Consider using `#6070FF` for the background color to match the Figma template.
 - Your design:
 - 
image

 - Figma template:
 - 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @jkanyi-web, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Form working correctly. ✔️
 - No linter errors. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by supreeti** [comment]
Change background color for contact form in style.css file as figma template.

File: style.css (hunk 1 above)
Comment (supreeti) [reply style.css]: change done as figma template

**Review by emmiiorji — APPROVED** [review]
Hi @supreeti,

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Adhered to GitHub flow
- Requested change has been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**